deps/wazuh_testing/wazuh_testing/mocking/__init__.py

```diff
@@ -104,6 +104,8 @@
                'name': 'debian10'},
     'STRETCH': {'os_name': 'Debian GNU/Linux', 'os_major': '9', 'os_minor': '0', 'os_platform': 'debian',
                 'name': 'debian9'},
+    'JAMMY': {'os_name': 'Ubuntu', 'os_major': '22', 'os_minor': '04', 'os_platform': 'ubuntu',
+              'name': 'Ubuntu', 'os_version': '22.04 (Jammy Jellyfish)', 'os_codename': 'jammy', 'os_arch': 'x86_64'}
```

Comment:
Move JAMMY after the FOCAL one

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2957/commits/aa779206c09b04849364bf70fc75c296425cba26